File: 6sem_curr/Mobile_Aps/lab01/task_1/in_java/src/main/java/com/example/l1z1_java/MainActivity.java
package com.example.l1z1_java;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;
import android.widget.Toast;

import java.util.Random;

public class MainActivity extends AppCompatActivity {
    private int score = 0;
    private int val1 = 0;
    private int val2 = 0;
    private Random generator = new Random();

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        roll();
    }

    private void roll() {
        TextView scoreInfo = (TextView) findViewById(R.id.points);
        scoreInfo.setText("Punkty: " + score);
        val1 = generator.nextInt(100);
        val2 = generator.nextInt(100);
        TextView leftBtn = (TextView) findViewById(R.id.leftButton);
        TextView rightBtn = (TextView) findViewById(R.id.rightButton);
        leftBtn.setText("" + val1);
        rightBtn.setText("" + val2);
    }

    public void rightClick(View view) {
        if (val1 <= val2) {
            score++;
            Toast.makeText(this, "Dobrze!!!", Toast.LENGTH_SHORT).show();
        }
        else {
            score--;
            Toast.makeText(this, "Źle!!!", Toast.LENGTH_SHORT).show();
        }
        roll();
    }

    public void leftClick(View view) {
        if (val1 >= val2) {
            score++;
            Toast.makeText(this, "Dobrze!!!", Toast.LENGTH_SHORT).show();
        }
        else {
            score--;
            Toast.makeText(this, "Źle!!!", Toast.LENGTH_SHORT).show();
        }
        roll();
    }
}

File: 6sem_curr/Mobile_Aps/lab01/task_1/in_kotlin/src/main/java/com/example/l1z1_kotlin/MainActivity.kt
package com.example.l1z1_kotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TextView
import android.widget.Toast
import kotlin.random.Random

class MainActivity : AppCompatActivity() {
    private var score: Int = 0
    private var val1: Int  = 0
    private var val2: Int  = 0

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        roll()
    }

    private fun roll() {
        findViewById<TextView>(R.id.points).text = "Punkty: $score"
        val1 = Random.nextInt(100)
        val2 = Random.nextInt(100)
        findViewById<TextView>(R.id.leftButton).text  = "$val1"
        findViewById<TextView>(R.id.rightButton).text = "$val2"
    }

    fun leftClick(view: View) {
        if (val1 >= val2) {
            score++
            Toast.makeText(this, "Dobrze!!!", Toast.LENGTH_SHORT).show()
        }
        else {
            score--
            Toast.makeText(this, "Źle!!!", Toast.LENGTH_SHORT).show()
        }
        roll()
    }

    fun rightClick(view: View) {
        if (val1 <= val2) {
            score++
            Toast.makeText(this, "Dobrze!!!", Toast.LENGTH_SHORT).show()
        }
        else {
            score--
            Toast.makeText(this, "Źle!!!", Toast.LENGTH_SHORT).show()
        }
        roll()
    }
}
